UnitsNet/Scripts/Include-GenerateQuantitySourceCode.ps1

```diff
@@ -1010,9 +1024,29 @@ function GenerateConversionMethods([GeneratorArgs]$genArgs)
             return ToUnit(unitAs$unitEnumName);
         }
 
+        /// <summary>
+        ///     Converts the quantity into a quantity which has units compatible with the given <see cref="UnitSystem"/>.
```

Comment:
Grammar nazi 🎈 , but `which` should always be preceded with a comma.
I also changed the descriptions a bit in IQuantity, so after you review those then match this xmldoc up with those and regen the files.

Reply:
Pinging this one, please review my xmldoc revisions in IQuantity, and make this consistent with that, then regen.

Reply:
Looks good to me. I just did an inheritdoc to keep things cleaner.